app/screens/DrawerScreen/index.js

```diff
@@ -88,7 +88,7 @@ class DrawerScreen extends Component {
            {/*for appstore app*/}
         {/* <Text style={this.styles.versionText}>APP VERSION {this.state.currentVersion}</Text> */}
         { /*//for tesing */}
-        <Text style={this.styles.versionText}>APP VERSION 1.1.0h</Text>
+        <Text style={this.styles.versionText}>APP VERSION 1.0.0</Text>
```

Comment:
Why is the version reverted to 1.0.0 ?

Reply:
This is because I thought this version number is for test-apk only,but if you if to be 1.1.0h I'll do that.